dqc_us_rules/__init__.py

```diff
@@ -59,7 +59,7 @@ def _plugins_to_run(mod, include_start=True):
 
 __pluginInfo__ = {
     'name': 'DQC.SEC.ALL',
-    'version': '3.0',
+    'version': '3.2',
```

Comment:
Yes.  I had not remembered to change the plugin version when I updated the rule version.  This change gets them in sync.